modules/cli/src/main/scala/scala/cli/commands/BspOptions.scala

```diff
@@ -2,31 +2,21 @@ package scala.cli.commands
 
 import caseapp._
 
-import scala.build.options.BuildOptions
+import java.nio.file.Path
 
 // format: off
 final case class BspOptions(
   // FIXME There might be too many options in SharedOptions for the bsp command…
   @Recurse
-    shared: SharedOptions = SharedOptions()
+    shared: SharedOptions = SharedOptions(),
+  
+  @Group("Compilation server")
+  @HelpMessage("Path to a json file containing BSP configuration")
```

Comment:
It's not a "BSP configuration". It's more command-line options.
```suggestion
  @HelpMessage("Command-line options JSON file")
```